book/src/road-map.md

```diff
@@ -0,0 +1,33 @@
+### Vendor Management
+
+- [ ] Manual SBOM upload screen (under development)
+- [ ] Role-based permissions related to SBOM upload screen (under development)
+- [ ] Vendor SBOM staleness/freshness policy tracking and enforcement (research & development)
+- [ ] Machine to machine vendor upload mechanism (research)
+
```

Comment:
Should we put Report generation in there too?

Reply:
🤦  Yeah, we should totally have an analytics section, but I'm not sure about what details to add there.

Reply:
I added something here.  Please take another look when you get a second.

Reply:
I should probably write that.  I can start it with this analytic.  Actually, updating this document should be part of Definition of Done, no?  We need to add to DoD for this and the SDL Spec. if necessary.